src/renderer/components/Media.js

```diff
@@ -53,28 +53,25 @@ const MediaGalleryItem = styled.div`
 
 const GROUPS = {
   images: {
-    key: 'Images',
     values: [C.DC_MSG_GIF, C.DC_MSG_IMAGE]
   },
-  videos: {
-    key: 'Videos',
+  video: {
     values: [C.DC_MSG_VIDEO]
   },
   audio: {
-    key: 'Audio',
     values: [C.DC_MSG_AUDIO, C.DC_MSG_VOICE]
   },
   documents: {
-    key: 'Documents',
     values: [C.DC_MSG_FILE]
   }
 }
```

Comment:
Lets just use the keys from `GROUPS` object. No need to add extra keys which are essentially identical.